docs/lib/auth/working-with-api.md

```diff
@@ -1,7 +1,14 @@
 ---
-title: Working with API
+title: Working with other Category
 description: Learn more about how to use the Amplify Framework's auth APIs 
 ---
 
-<inline-fragment platform="ios" src="~/sdk/auth/fragments/ios/working-with-api.md"></inline-fragment>
-<inline-fragment platform="android" src="~/sdk/auth/fragments/android/working-with-api.md"></inline-fragment>
\ No newline at end of file
+How to use auth category with other categories
+
+## UnAuth access
+
+## Get aws credentials
+
+## Get identity id
+
+## Get cognito tokens
```

Comment:
This needs to be removed, otherwise, JS will show a section called "Working with other category"